docs/interface.md

```diff
@@ -30,9 +30,9 @@ partial interface WorkerNavigator {
 };
 
 dictionary ShareData {
-  DOMString? title;
-  DOMString? text;
-  DOMString? url;
+  DOMString title;
+  DOMString text;
+  USVString url;
```

Comment:
I'm struggling to understand why we use USVString for URLs and DOMString for other things.

Looking at the [WebIDL spec](https://heycam.github.io/webidl/#idl-USVString), it looks like DOMString is a sequence of 16-bit code units (i.e., a UTF-16 string that can have surrogate encoding errors) and a USVString is a sequence of Unicode code points (i.e., a string of unspecified encoding that has to be valid).

USVString seems like a desirable property for any string attribute, not just URLs. Why the inconsistency?